static/js/common/upload-addon.js

```diff
@@ -382,7 +382,7 @@
                                 $upload_field.trigger("upload_finished", [file]);
                             }
                         });
-                    }, 1000);
+                    }, 5000);
```

Comment:
if a submission is quick, does this mean they'd have to wait at least 5 seconds to continue? That seems sort of long. We have enough caching that I'm not worried about hitting this URL so often.